src/website/views/components/picker-group.pug

```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
       </label>
     </div>
 
+h3 Texts and icons
+p.-text Rendering picker group elements with icons <code>&lt;i class="a-icon icon-atom"&gt;&lt;/i&gt;</code>.
+.example.-mb--3
+  .-p--3
+    .m-pickerGroup
+      input.a-picker(type="radio", name='radios', id=`ib-radio1`)
+      label(for=`ib-radio1`)
+        i.a-icon.icon-atom
+        span
+          = `Radio1`
+      input.a-picker(type="radio", name='radios', id=`ib-radio2`)
+      label(for=`ib-radio2`)
+        span
+          = `Radio2`
+        i.a-icon.icon-atom
+      input.a-picker(type="radio", name='radios', id=`ib-radio3`)
+      label(for=`ib-radio3`)
+        i.a-icon.icon-atom
+        span
+          = `Radio3`
```

Comment:
@janibolkvadze can you remove this line and move to line 52 with this format span Radio 3